api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -374,11 +374,30 @@ type NodeAgentCommonFields struct {
 	PodConfig *PodConfig `json:"podConfig,omitempty"`
 }
 
-type RestorePVC struct {
-	// IgnoreDelayBinding indicates to ignore delay binding the restorePVC when it is in WaitForFirstConsumer mode
-	IgnoreDelayBinding bool `json:"ignoreDelayBinding,omitempty"`
+// LoadConcurrency is the config for data path load concurrency per node.
+// Struct should be same as:
+// https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L52-L58
+type LoadConcurrency struct {
+	// GlobalConfig specifies the concurrency number to all nodes for which per-node config is not specified
+	GlobalConfig int `json:"globalConfig,omitempty"`
+
+	// PerNodeConfig specifies the concurrency number to nodes matched by rules
+	PerNodeConfig []RuledConfigs `json:"perNodeConfig,omitempty"`
 }
 
+// LoadAffinity is the config for data path load affinity.
+// Used by the Node-Agent, that needs to match the DataMover and the RepositoryMaintenance pods.
+// Struct should be same as:
+// https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L60-L63
```

Comment:
it should actually points to this https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L95 (which is the same object, do not know why they duplicated it)

Reply:
Yeah it was moved in upstream to use from kube https://github.com/openshift/velero/commit/26cc41f26daad20bd9a2c87ae408c240b28e1165#diff-314c2d754eaafb3e32404e38549140072c0b258108b1befe8804c6ca5978bdb0

Reply:
Checked this and it is correctly pointing to:
https://github.com/openshift/velero/blob/584cf1148a746838ee67aa27e3e4e0ded1f5c069/pkg/nodeagent/node_agent.go#L60-L63

We can't use directly the L95 due to the fact the type *kube.LoadAffinity has no field or method DeepCopyInto.